client/src/app/styles/_variables.scss

```diff
@@ -0,0 +1,7 @@
+// Screen sizes
+$tiny: 360px;
+$small: 640px;
+$medium: 800px;
+$normal: 1024px;
+$large: 1200px;
+$xlarge: 1440px;
```

Comment:
start with $small and add extra $xxlarge variable.
convention of this variables is to keep as cloaths sizes so: s,m,l,xl, xxl etc